Added solution for Operation Spicy Screenshot Supremacy

Kshitij-K-Singh opened the pull request.

Abhishek-Punhani commented:
Overall Good Job

Abhishek-Punhani commented:
Further If you wanna try can try making a webpage just make html ,css files and use this js file as script to html file and take input from the user the urls and provide an option to download the pdf

Kshitij-K-Singh commented:
Bhaiya did following changes: 
 1.Modified the Javascript for the user inputs
 2. added index.html and styles.css for the website
 3. A preview file for the working
